content/rosa/entra-external-auth/index.md

```diff
@@ -0,0 +1,467 @@
+
+```bash
+NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+console                                    4.15.43   True        False         False      7m3s
+csi-snapshot-controller                    4.15.43   True        False         False      117m
+dns                                        4.15.43   True        False         False      109m
+image-registry                             4.15.43   True        False         False      109m
+ingress                                    4.15.43   True        False         False      109m
+insights                                   4.15.43   True        False         False      110m
+kube-apiserver                             4.15.43   True        False         False      117m
+kube-controller-manager                    4.15.43   True        False         False      117m
+kube-scheduler                             4.15.43   True        False         False      117m
+kube-storage-version-migrator              4.15.43   True        False         False      110m
+monitoring                                 4.15.43   True        False         False      108m
+network                                    4.15.43   True        False         False      117m
+node-tuning                                4.15.43   True        False         False      111m
+openshift-apiserver                        4.15.43   True        False         False      117m
+openshift-controller-manager               4.15.43   True        False         False      117m
+openshift-samples                          4.15.43   True        False         False      109m
+operator-lifecycle-manager                 4.15.43   True        False         False      117m
+operator-lifecycle-manager-catalog         4.15.43   True        False         False      117m
+operator-lifecycle-manager-packageserver   4.15.43   True        False         False      117m
+service-ca                                 4.15.43   True        False         False      110m
+storage                                    4.15.43   True        False         False      111m
+```
+
+### Grant cluster-admin permissions to individual groups
+By default, OpenShift does not grant permissions to take any action inside of your cluster when a user first logs in. ROSA includes a significant number of preconfigured roles, including the `cluster-admin` role that grants full access and control over the cluster. The cluster does not automatically create `RoleBindings` and `ClusterRoleBindings` for the groups that are included in your access token; you are responsible for creating those bindings by using your own processes.
+
+In this example, we will grant a group access to the `cluster-admin` role by creating a ClusterRoleBinding to the group ID.
+
+To retrieve the Azure Entra Group ID, search for Entra Group ID and copy the Object ID.
+
+![Azure Portal - Get Group ID](./images/azure-portal_groups_id.png)
```

Comment:
This is an active group and the screenshot should be of a dummy group. Please replace.